src/components/MapListCard.tsx

```diff
@@ -49,20 +51,32 @@ const MapListCard = ({
     (metric) => metric.name == "English Language Learners",
   );
 
+  const learnMoreRef = useRef<HTMLAnchorElement>(null);
+
   const handleTransitionEnd = (e: React.TransitionEvent<HTMLDivElement>) => {
     if (e.propertyName === "max-height" && isExpanded) {
       cardRef.current?.scrollIntoView({ behavior: "smooth", block: "nearest" });
     }
   };
 
   function onClick(e: React.MouseEvent<HTMLDivElement>) {
+    if (learnMoreRef.current && learnMoreRef.current.contains(e.target as Node)) {
```

Comment:
this was causing a setSelectedSchool(null), which we don't want because then we lose the context